### PR TITLE
Enterprise Release on January 2019

### PR DESCRIPTION
**Pull request by ybiquitous**
Add a release note and some changes about Enterprise release on **January 2019**.

### CODE REVIEW

**Comment by yykamei** [comment]
I'll do force-push in this branch for January 2019 release.

**Comment by yykamei** [comment]
Changes from the previous release:

* [sideci](https://github.com/sider/sideci/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Amerged+created%3A2018-12-10..2019-01-10+-label%3Adependencies+)
* [catpost](https://github.com/sider/catpost/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Amerged+created%3A2018-12-10..2019-01-10+-label%3Adependencies+)
* [setaria](https://github.com/sider/setaria/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Amerged+created%3A2018-12-10..2019-01-10+-label%3Adependencies+)

File: on-premises/releases/README.md [hunk 1]
```diff
@@ -4,9 +4,31 @@
...
+| catpost          | release-2019011001 |
+| setaria          | release-2019011001 |
+
+## release-2019011001
+
+### Features
+
+* Update the Admin Dashboard page to show more detailed statistics
+* Redesign the Repository Settings page
+* Redesign the display of issues on a Pull Request page
```
Comment (wata727) [inline on-premises/releases/README.md]: GitHub App support
https://blog.sideci.com/you-can-now-install-sider-as-a-github-app-f52a073b54b7

File: on-premises/releases/README.md [hunk 2]
```diff
@@ -4,9 +4,33 @@
...
+
+### Notes
+
+* Drop support for translation of issues (Fore more information, see [here](https://blog-ja.sideci.com/entry/20181203-announsement))
+
+### Update Procedure
+
+This release requires you to configure additional settings.
+First, follow the [guide](../../on-premises/quick-start/setup.md#creating-a-new-github-app-for-sider) to create a GitHub App on your GitHub Enterprise.
+After that, update Sider Enterprise as usual (See [here](../../on-premises/quick-start/update.md)).
```
Comment (wata727) [inline on-premises/releases/README.md]: I think that we should explain about the new required environment variables `ACTION_MAILER_DEFAULT_FROM_EMAIL`.

Comment (yykamei) [reply on-premises/releases/README.md]: You're right. I forgot it. Thanks.

File: on-premises/releases/README.md [hunk 3]
```diff
@@ -4,9 +4,37 @@
...
+| setaria          | release-2019011001 |
+
+## release-2019011001
+
+### Features
+
+* Update the Admin Dashboard page to show more detailed statistics
+* Redesign the Repository Settings page
+* Redesign the display of issues on a Pull Request page
+* Add suport for GitHub Apps (see [here](https://blog.sideci.com/you-can-now-install-sider-as-a-github-app-f52a073b54b7))
```
Comment (ybiquitous) [inline on-premises/releases/README.md]: [suggestion] How about `here` -> `our blog`? The word `here` seems ambigous (this documentation or somewhere...)

File: on-premises/releases/README.md [hunk 4]
```diff
@@ -4,9 +4,37 @@
...
+
+### Fixes
+
+* Bug fixes
+* Remove `Account & Billing` from Organization Settings page because it's not necessary for Sider Enterprise
+
+### Notes
+
+* Drop support for translation of issues (Fore more information, see [here](https://blog-ja.sideci.com/entry/20181203-announsement))
```
Comment (ybiquitous) [inline on-premises/releases/README.md]: [suggestion] How about `Fore more information` → `For more details` ?

**Comment by ybiquitous** [comment]
Looks good! 👍 

I'm this PR author, so I can't approve it 😅

**Comment by yykamei** [comment]
OK, I'll merge after rebasing.

File: on-premises/releases/README.md [hunk 5]
```diff
@@ -32,7 +32,8 @@ This release requires you to configure additional settings.
 
 First, follow the [guide](../../on-premises/quick-start/setup.md#creating-a-new-github-app-for-sider) to create a GitHub App on your GitHub Enterprise.
 
-Secondly, set the environment variable `ACTION_MAILER_DEFAULT_FROM_EMAIL` in `sideci.env` file. For example, you can set it `support@example.com`. This email address is used when sending system error. See [here](../../on-premises/quick-start/setup.md).
+Secondly, set the environment variable `ACTION_MAILER_DEFAULT_FROM_EMAIL` in `sideci.env`, `catpost.env`, and `setaria.env` file.
```
Comment (yykamei) [inline on-premises/releases/README.md]: Users must update all environment files.

**Pull request merged**